src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -74,7 +83,7 @@ export class BidResult extends React.Component<BidResultProps> {
                 }}
               />
             ) : (
-              <BidGhostButton text="Continue" onPress={() => null} />
+              <BidGhostButton text="Continue" onPress={this.exitBidFlow} />
```

Comment:
I was wondering if there were problems with this syntax, binding to `this` or anything?